ui/desktop/app/templates/scopes/scope/projects/settings/index.hbs

```diff
@@ -3,4 +3,54 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{outlet}}
\ No newline at end of file
+<SettingsCard @header={{t 'titles.application'}} @icon='boundary'>
+  <div class='content'>
+    <div class='app-information'>
+      <div>
+        <Hds::Text::Display>
+          {{t 'titles.version'}}
+        </Hds::Text::Display>
+        <Hds::Text::Body @tag='p' @weight='medium'>
+          {{@model.desktopVersion}}
+        </Hds::Text::Body>
+      </div>
+      <div>
+        <Hds::Text::Display>
+          {{t 'titles.cli'}}
```

Comment:
Should this say CLI version?

Reply:
[figma](https://www.figma.com/design/28zD8tKqa9yUQVO13THm77/Transparent-Sessions-%26-Aliases?node-id=4623-123988&t=Eub6bjk8XC75GRHr-0) says CLI and cache daemon.  

@joshuaogle thoughts?  [this would be the headers in the application info section]

Reply:
discussed offline and updated based on design feedback to use prefixes